File: app/src/main/java/com/scullyapps/spoonsbottleup/MainActivity.java
package com.scullyapps.spoonsbottleup;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.content.Intent;
import android.database.sqlite.SQLiteDatabase;
import android.os.Bundle;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.Toast;

import com.scullyapps.spoonsbottleup.database.BottleDatabase;

import butterknife.BindView;
import butterknife.ButterKnife;

public class MainActivity extends AppCompatActivity {

    @BindView(R.id.mmenu_button_settings) Button btnSettings;
    @BindView(R.id.mmenu_button_exit)     Button btnExit;
    @BindView(R.id.mmenu_button_start)    Button btnStart;

    @BindView(R.id.mmenu_about_button)
    LinearLayout btnAbout;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ButterKnife.bind(this);

        ActionBar toolbar = getSupportActionBar();
        toolbar.hide();


        //        db.updateListOrder(0,0);

        setupButtons();
    }

    private void setupButtons() {

        Intent start = new Intent();

        btnStart.setOnClickListener(v -> {
            start.setClass(this, CountActivity.class);
            startActivity(start);
        });

        btnSettings.setOnClickListener(v -> {
            start.setClass(this, SettingsActivity.class);
            startActivity(start);
        });

        btnAbout.setOnClickListener(v -> {
            Toast.makeText(this, "Implement me!", Toast.LENGTH_SHORT).show();
        });

        btnExit.setOnClickListener(v -> {
            System.exit(0);
        });


    }
}

File: app/src/main/java/com/scullyapps/spoonsbottleup/activities/MainActivity.kt
package com.scullyapps.spoonsbottleup.activities

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.LinearLayout
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import butterknife.BindView
import butterknife.ButterKnife
import com.scullyapps.spoonsbottleup.R
import kotlinx.android.synthetic.main.activity_main.*
import kotlin.system.exitProcess

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        supportActionBar?.hide()

        // db.updateListOrder(0,0);

        setupButtons()
    }

    private fun setupButtons() {
        val start = Intent()
        mmenu_button_start.setOnClickListener {
            start.setClass(this, CountActivity::class.java)
            startActivity(start)
        }
        mmenu_button_settings.setOnClickListener {
            start.setClass(this, SettingsActivity::class.java)
            startActivity(start)
        }
        mmenu_about_button.setOnClickListener { Toast.makeText(this, "Implement me!", Toast.LENGTH_SHORT).show() }
        mmenu_button_exit.setOnClickListener { exitProcess(0) }
    }
}